README.md

```diff
@@ -1,2 +1,48 @@
-# text-editor
-Development Text editor. No guarantees are given that this works or keeps working.
+# Text Editor
+Rich inline text editing
+
+## Features
+* Edit selected text
+
+## Dependencies
+Mendix 7.1
+
+## Demo project
+http://texteditorwidget.mxapps.io
+
+## Usage
+Place the widget in the context of an object that has a string attribute.
```

Comment:
Place the widget in a data view, list view or template grid with a data source that has a string attribute.